src/instructlab/config/init.py

```diff
@@ -116,81 +118,15 @@ def init(
         click.echo(
             f"\nGenerating config file and profiles:\n    {DEFAULTS.CONFIG_FILE}\n    {DEFAULTS.TRAIN_PROFILE_DIR}\n"
         )
-        recreate_train_profiles(overwrite=True)
+        recreate_system_profiles(overwrite=True)
     else:
         click.echo(f"\nGenerating config file:\n    {DEFAULTS.CONFIG_FILE}\n")
     if train_profile is not None:
```

Comment:
Why is there still mention of train profiles here and on line 117 above? Are we keeping them around for another release or totally deprecating them with this PR.

Reply:
all instances of train_profile should be removed now